content/en/Platform Deep Dive/Pentests/Reports/_index.md

```diff
@@ -12,16 +12,23 @@ Download a pentest report to view security issues that our pentesters found.
 
 You can download a report for a pentest once we move it to the Remediation [state](/penteststates/). Report types and their [contents](/platform-deep-dive/pentests/reports/report-contents/) differ for each [pentest type](/platform-deep-dive/pentests/pentest-types/).
 
+To download a report as a PDF file:
+
+1. On the pentest page, navigate to **Reports**.
+1. For [Comprehensive Pentests](/getting-started/glossary/#comprehensive-pentest), select the report type.
+   - {{% customize-report-tier %}}
+1. Select **Download**.
+
 ## Pentest Report Types
 
 | Report Type | Available for [Pentest Type](/platform-deep-dive/pentests/pentest-types/) | Description | [Customizable](/platform-deep-dive/pentests/reports/customize-report/) |
 |-----|-----|-----|-----|
-| Automated Report | Agile Pentest | A system-generated report for an [Agile Pentest](/getting-started/glossary/#agile-pentest) intended for internal use. Includes the following sections:<br><br>- Pentester user information<br>- Executive Summary<br>- Methodology<br>- Post-Test Remediation<br>- Finding Details | No |
-| Customer Letter | Comprehensive Pentest | An executive summary of the pentest. May be used as a certificate of completion. Great for external shareholders. Includes:<br>  <br>- Executive Summary <br>- Methodology | Yes |
-| Attestation Report | Comprehensive Pentest | Adds the following information to the customer letter: <br> <br>- Pentester user information <br>- An overall list of findings                                                                     | Yes |
-| Attestation Letter | Comprehensive Pentest | Includes the [executive summary](./report-contents/#executive-summary) as a formal letter, suitable for external stakeholders or customers                                                                     | No |
-| Full Report | Comprehensive Pentest | Includes the following report sections, beyond attestation:<br>  <br>- Executive Summary <br>- Scope of Work <br>- Methodology <br>- Summary of Findings <br>- Recommendations <br>- Post-Test Remediation     | Yes |
-| Full Report + Finding Details | Comprehensive Pentest | Adds details of every test finding to the full report. Details include:<br>  <br>- Vulnerability Type <br>- Description <br>- Proof of Concept <br>- Severity <br>- Suggested Fix     | Yes |
+| Automated Report | Agile Pentest | {{% automated-report %}} Includes the following sections:<br><br>- Pentester user information<br>- Executive Summary<br>- Methodology<br>- Post-Test Remediation<br>- Finding Details | No |
+| Customer Letter | Comprehensive Pentest | {{% customer-letter %}} Includes:<br>  <br>- Executive Summary <br>- Methodology | Yes |
+| Attestation Report | Comprehensive Pentest | {{% attestation-report %}}<br><br>Adds the following information to the Customer Letter: <br> <br>- Pentester user information <br>- An overall list of findings  | Yes |
```

Comment:
@TheInstaGrahame Hi Grahame, I need your help with reviewing some content related to reports.

On this [page](https://developer.cobalt.io/platform-deep-dive/pentests/reports/), we state the an Attestation Report includes the following:
- Pentester user information 
- An overall list of findings (I can't see it in the app)

What is the real difference between a Customer Letter and Attestation Report?

I added definitions for all report types to `layouts/shortcodes`. Can you please review? Thank you.

Reply:
Hey!

So, for context, the Attestation Report is what we had before we invented the Attestation Letter. THe request we had from customers was a one-page letter indicating that we had performed a test. 

That said, the Attestation Report includes: 
- Executive Summary
- Executive Analysis + Finding Breakdown
- Methodology/Rating Info

The Customer Letter has:
- Executive Summary
- Methodology/Rating Info

And just to round out the trio, Attestation Letter has:
- Executive Summary

The Executive Summary section of the report is essentially two paragraphs that say "Cobalt came and performed X testing. We had Y testers, and did it from A Date to B Date."

The Analysis includes info about the test itself, at a high level.

The methodology and rating info is boilerplate, and mostly serves to explain our rating system.

(FYI: Product is currently working on an update that will put the Finding Breakdown table into the Attestation Letter, and make it an option for customers to show or hide. That's not quite done yet, but I'll keep you and Mike posted on it)

Reply:
Thank you @TheInstaGrahame, I made changes based on our discussion in Slack.